test/model/minister-model.js

```diff
@@ -1,4 +1,4 @@
-const { baseModel } = require('./../../src/base/baseModel');
+const { baseModel } = require('@venncity/sequelize-model/src/base/baseModel');
```

Comment:
isn't the baseModel exported from the entry file?
wouldn't const { baseModel } = require('@venncity/sequelize-model') work?
if not, lets make it work, its not healthy to directly access other module src

Reply:
you are right, I moved the file and webstorm automatically refactored the requires